google/api/field_behavior.proto

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Google LLC
```

Comment:
So this and the other protos in this directory are copied from elsewhere, correct?

I would, at the very least, add a README or modify the top-level comments to say where they were copied from.

Ideally (I think....) I would depend on the source repos and use the protos from there rather than having local copies. What was your reason for not doing that?

Reply:
This is for convenience. Those protos are not requried for the converter to function. I use them to validate the correctness of the generated proto (i.e. without the includes, I get errors of it not being valid). 

Maybe I should just remove those. I'll probably do those once this component is productionalized.

Reply:
OK. Could you file a bug to track that?